kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -187,6 +191,29 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
     buildBatch(sessionManager.getBatchSessionImpl(sessionHandle))
   }
 
+  @ApiResponse(
+    responseCode = "200",
+    content = Array(new Content(
+      mediaType = MediaType.APPLICATION_JSON,
+      schema = new Schema(implementation = classOf[Batch]))),
+    description = "create and open a batch session with uploading resource file")
+  @POST
+  @Path("/upload")
+  @Consumes(Array(MediaType.MULTIPART_FORM_DATA))
+  def openBatchSessionWithUpload(
+      @FormDataParam("batchRequest") batchRequestStr: String,
+      @FormDataParam("resourceFile") resourceFileInputStream: InputStream,
+      @FormDataParam("resourceFile") resourceFileMetadata: FormDataContentDisposition): Batch = {
```

Comment:
is it duplicated？

Reply:
I mean there are two FormDataParams with the same name `resourceFile `.

Reply:
Seems both are required, as instruction from Jerysey's doc here(https://eclipse-ee4j.github.io/jersey.github.io/documentation/latest/media.html#d0e9649).
One is for getting input stream and the other for uploaded file's metadata.